### DISCUSSION TITLE
Automatically delete past events?

### DISCUSSION THREAD

**Discussion by nofishonfriday** [FossifyOrg/General-Discussion#468] [Q&A] [answered]
As per title, is there a way to automatically delete past events?

Background:
I noticed only recently by chance looking at the 'Simple event list', all past events are listed there, and it accumulates over time...

**Comment by Aga-C** [comment, marked as answer]
No, but you can create a [feature request](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.
